spec/requests/can_respond_with_collection_of_latest_experiences_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe 'GET /api/articles', type: :request do
+  let!(:test_story) { create(:article, title: "Story Article") }
+  let!(:test_article1) { create(:article, article_type: 0, created_at: Time.current - 5.days) }
+  let!(:test_article2) { create(:article, title: 'Test Article 2', article_type: 0, created_at: Time.current - 2.days) }
+  let!(:test_article3) { create(:article, article_type: 0, created_at: Time.current - 3.days) }
+  let!(:test_article4) { create(:article, article_type: 0, created_at: Time.current - 4.days) }
+  let!(:test_article5) { create(:article, title: 'Test Article 5', article_type: 0, created_at: Time.current - 1.days) }
+  describe 'successfully' do
+    before do
+      get '/api/articles',
+          params: {
+            article_type: 'experience'
```

Comment:
think about what we experienced how axios works that you have to send this is a query param with a GET request. Functionality wise this won't change anything because rails is set up the way that it accesses params anyway, but you test should reflect on how you will send this request to the API.